templates/user/base.html

```diff
@@ -31,11 +31,11 @@
 
               <!-- Link -->
               <li class="nav-item">
-                <a class="nav-link" href="/home/">Home</a>
+                <a class="nav-link" href="/">Home</a>
```

Comment:
```suggestion
                <a class="nav-link" href="{% url 'user:home" %}>Home</a>
```
のようにテンプレートでも参照できます。変更に備えてこうしておくのがおすすめです！（前回のコメント抜けです，すみません）